filesystem/util.go

```diff
@@ -94,7 +109,20 @@ func Unmount(device, target string) error {
 			return nil
 		}
 
-		switch err := unix.Unmount(target, unix.UMOUNT_NOFOLLOW); err {
+		// Handle EINTR signal for Go 1.14+
+		var err error
+		for {
+			err = unix.Unmount(target, unix.UMOUNT_NOFOLLOW)
+			if err == nil {
+				break
+			}
+			if e, ok := err.(temporaryer); ok && e.Temporary() {
+				continue
+			}
+			return err
```

Comment:
If the err is not temporary, the return statement skips switch block.
I think use break statement.

Reply:
@toshipp Sorry...I fixed it.